src/types/dto/review.ts

```diff
@@ -13,7 +17,7 @@ export interface EditedItemDto {
   lastEditedTime: number
 }
 
-export interface UserDto {
+export interface UserDto extends Jsonify<User> {
   email: string
   role: CollaboratorRoles
 }
```

Comment:
`UserDto` still doesn't seem to recognise `User` fields - is this the intended behaviour?

Reply:
double checking - what do you mean by this

Reply:
resolved offline - meant that intellisense wasn't showing the properties. 

as to why this happens, i suspect this is because of the `class` keyword. using `ConditionalExcept<User, Function>` will extract the properties but then we'd need to hint the type of `toJSON` properly also.

went with just specifying props manually as it seemed like less effort ._.